packages/elements/src/tree/elements/tree.ts

```diff
@@ -20,8 +20,8 @@ const EXPAND_TOGGLE_ATTR = 'expand-toggle';
  * Displays a tree structure
  * to be used for menus and group selections
  *
- * @fires value-changed - Fired when the users changed selection item.
- * @fires expanded-changed - Fired when an item's expanded state has changed.
+ * @fires value-changed - Fired when the user commits a value change. The event is not triggered if `value` property is changed by programmatically.
+ * @fires expanded-changed - Fired when the user expanded or collapsed item. The event is not triggered if `expanded` property is changed by programmatically.
```

Comment:
Fired when the user expand"s"~~ed~~ or collapse"s"~~d~~ item. he event is not triggered if `expanded` property is changed ~~by~~ programmatically.